bin/cron/applab_datasets/weather

```diff
@@ -5,10 +5,16 @@ require_relative '../../../deployment'
 require 'cdo/only_one'
 require 'net/http'
 require 'json'
+require 'ostruct'
+require 'date'
 require_relative '../../../shared/middleware/helpers/firebase_helper'
 
 WeatherForecastOffice = Struct.new(:city, :state, :zip)
-WeatherRecord = Struct.new(:id, :city, :state, :timestamp, :temp_min, :temp_max, :weather, :icon, :wind_speed)
+
+def cardinal_dir_from_degrees(degrees)
+  dirs = %w(N NE E SE S SW W NW N)
```

Comment:
Is it a problem that `N` is in here twice?

Reply:
Oh, I see. For 0 and 360.

Reply:
Ok, second question. Is it ever possible to get degrees above 360 or below 0?

Reply:
Yes, intentional for `N` twice, for 0 and 360 as you said :) 
I don't think the API response will ever have numbers outside that range.

Reply:
Ok. What about error input? If degrees is "nil" or something similar, will it cause the script to error out and exit early? Is that ok?

Reply:
✔️ Nice update